linter/updates.go

```diff
@@ -170,14 +170,14 @@ func (u *UpdatesLinter) RunWithContext(ctx context.Context, roles <-chan require
 			results <- Result{
 				Role:     r,
 				Level:    LevelWarning,
-				Msg:      fmt.Sprintf("%s: unable to find %s between the available versions, tag a new release or use %s", roleName, r.Version, latest),
+				Msg:      fmt.Sprintf("%s unable to find [%s] between the available versions, tag a new release or use [%s]", roleName, r.Version, latest),
```

Comment:
I personally don't like the square brackets cause in my mind these recall the concept of list.
Do you think it makes sense listing all the available versions here? I am afraid this would be too much verbose.

If not, I would roll-back to the original message

```suggestion
				Msg:      fmt.Sprintf("%s: unable to find %s between the available versions, tag a new release or use %s", roleName, r.Version, latest),
```

Reply:
I don't think it's necessary to list all versions.
If is unable to find the role version, probably we selected the wrong role in ansible galaxy